include/seqan/gff_io/gff_io_base.h

```diff
@@ -349,9 +349,12 @@ void readRecord(GffRecord & record, CharString & buffer, TFwdIterator & iter)
 {
     IsNewline isNewline;
 
-    clear(record);
+    // skip commented lines
+    while(!atEnd(iter) && value(iter) == '#')
```

Comment:
Oh.. I'm so sorry that you had to take your time for this simple change.
Perhaps your comments are longer than my modifications.

This is my first pull request to SeqAn, so I will be aware of coding conventions and others in the next time.
Thanks, again!